build/ClrInstrumentationEngine.cpp.props

```diff
@@ -34,11 +34,17 @@
     -->
   <PropertyGroup>
     <CodeAnalysisWarningsToDisable>/wd25007 /wd25021 /wd25031 /wd25046 /wd25090 /wd25129 /wd25164 /wd25165 /wd25166</CodeAnalysisWarningsToDisable>
+    <AdditionalWarningsToDisable>/wd4103</AdditionalWarningsToDisable>
```

Comment:
> <AdditionalWarningsToDisable>/wd4103</AdditionalWarningsToDisable> [](http://example.com/codeflow?start=3&length=67)

Undo this change. It was added because of a break in the C++ compiler.